src/main/java/frc/robot/commands/ramp/RampFollow.java

```diff
@@ -47,6 +50,8 @@ public void execute() {
 
     @Override
     public boolean isFinished() {
-        return alignable == null || alignableSupplier.get() == null || !alignable.equals(alignableSupplier.get());
+        boolean isFinished = alignable == null || alignableSupplier.get() == null || !alignable.equals(alignableSupplier.get());
+        SmartDashboard.putBoolean("RampDone", isFinished);
```

Comment:
if debug?

Reply:
Also - this would appear in logs no no real need

Reply:
removed